front50-core/src/main/groovy/com/netflix/spinnaker/front50/validator/GenericValidationErrors.groovy

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.front50.validator
+
+import org.springframework.validation.AbstractErrors
+import org.springframework.validation.Errors
+import org.springframework.validation.FieldError
+import org.springframework.validation.ObjectError;
+
+class GenericValidationErrors extends AbstractErrors {
```

Comment:
This is basically a copy of what we've got in `ApplicationValidationErrors` ... the two can likely be merged in a subsequent PR.

Reply:
Not sure it's really all that valuable to use Spring `Errors` but we've got this pattern in a few spots already.

Reply:
I think I'd prefer to use `Errors` over something else, personally. Converge on a standard, and one that we don't have to write 💯